### DISCUSSION TITLE
Transfer to ramworks dev group?

### DISCUSSION THREAD

**Discussion by DJshelfmushroom** [theshmunguss1/RamOS#12] [General]
@theshmunguss1 should we transfer this repo to ramworks? it is fully functional, it doesn't need a beta org anymore. It would also streamline workflows and allow for transferring issues/discussions.

**Comment by YesTheProxy** [comment]
I dissagree due to the fact that we did this so it would be unblocked at our school

**Comment by DJshelfmushroom** [comment]
really? how does that work?

**Reply by theshmunguss1** [reply]
I think our school is able to block domain extensions aswell. As in: [theshmunguss1.github.io](theshmunguss1.github.io) is blocked, but [github.io](github.io) is not.

**Reply by theshmunguss1** [reply]
I'm frankly flattered that the IT department hates me.

**Reply by YesTheProxy** [reply]
I'm flattered too, they've blocked my repos, ALL OF THEM
